src/wp-includes/block-supports/layout.php

```diff
@@ -834,7 +834,8 @@ function wp_render_layout_support_flag( $block_content, $block ) {
 			break;
 		}
 
-		if ( false !== strpos( $processor->get_attribute( 'class' ), $inner_block_wrapper_classes ) ) {
+		$class_attribute = $processor->get_attribute( 'class' );
+		if ( is_string( $class_attribute ) && false !== strpos( $class_attribute, $inner_block_wrapper_classes ) ) {
```

Comment:
Looks like you can make use of `str_contains()` here.

Reply:
+1 The polyfill version is available from 5.9. See https://github.com/WordPress/wordpress-develop/blob/6daf85302233bc99823b453691379be3a5eb5f22/src/wp-includes/compat.php#L468-L474

Reply:
Note to self: this will now require us to backport the usage of `str_contains` to the Gutenberg repo.

Reply:
[Backport to Gutenberg PR](https://github.com/WordPress/gutenberg/pull/58251)

Reply:
thanks for the update. generally I would encourage folks to think about separating the act of bringing over already-merged and accepted code from the act of changing it.

it can make it difficult to deal with when things go wrong because we've wrapped up a styling-preference change with what should otherwise be a somewhat mechanical copy operation.

another way this could have taken place is to create a PR in Gutenberg to update the code where it came from, and then port that change over, so we don't introduce divergence between the repos without a clear audit trail.

Reply:
That sounds like a better plan. I'll switch the approach and then target the new PR in Gutenberg.

Reply:
you're amazing. to be clear, I would have asked to change this PR if I thought it was important enough to reject, so thanks for going above and beyond to split these apart.

mostly I wanted us all to think about these kinds of things and the headaches that can happen when mangling them. "style-only changes" often introduce behavioral bugs and that's never fun in a crisis moment.